rwengine/src/script/ScriptMachine.cpp

```diff
@@ -12,11 +12,9 @@
 #include "script/ScriptModule.hpp"
 
 void ScriptMachine::executeThread(SCMThread& t, int msPassed) {
-    auto& players = getState()->world->players;
+    auto player = state->world->getPlayer();
 
-    if (!players.empty()) {
-        // @todo Add support for multiple players
```

Comment:
Maybe it's better not to remove this comment?

Reply:
Don't think it's possible to support multiple players anyway, because of the way how certain scm commands work. If we want to implement some sort of coop, missions have to be rewritten in some other scripting language.